### PR TITLE
P0 badges

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153232143-8957e10c-f126-4064-8711-6758e31bb71f.mp4

### CODE REVIEW

File: components/badges/badges.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="../../docs/components.css">
+    <link rel="stylesheet" href="../../landing page/style.css">
+    <link rel="stylesheet" href="../../docs/navigation.css">
+    <link rel="stylesheet" href="../../docs/sidebar.css">
```
Comment (Varun-28) [inline components/badges/badges.html:12]: you can import all CSS files in one file and import that file in your current CSS file. Rest looks good 👍

File: components/badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    margin-top: 40px;
+}
+
+
+
+.badge::after {
+    content: "12";
+    background-color: var( --complementary-light-green-color);
+    border-radius: 50%;
+    color: var( --text-secondry);
```
Comment (saiyerniakhil) [inline components/badges/badges.css:13]: not an issue, but typo's could be avoided ~secondry~ -> **secondary**

File: components/badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    position: absolute;
+    bottom: 25px;
+    left: 30px;
```
Comment (saiyerniakhil) [inline components/badges/badges.css]: when we are giving absolute units for positioning, when we're increasing or decreasing the object size, the placement might distort a bit. try to use relative units like `%`/`rem`

**Comment by saiyerniakhil** [comment]
The badge text is looking congested may be you try adding some padding to the text inside it so that it looks clean.
The rest looks good. Great work @stutisk 🎉

File: components/badges/badges.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+                <span class="bar"></span>
+                </a>
+
+            </div>
+            <div class="right-side">
+                <ul>
+                    <li><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+                    <li><a onclick="myFunction()" href="#"><i class="fa fa-moon-o" aria-hidden="true"></i></a></li>
```
Comment (abhij1607) [inline components/badges/badges.html]: Using event listener instead of onclick is a better approach

**Review by abhij1607 — APPROVED** [review]
Looks great!

File: components/badges/badges.css (hunk 2 above)
Comment (stutisk) [reply components/badges/badges.css:13]: oh thanku for pointing it out ! will make the changes required

File: components/badges/badges.html (hunk 4 above)
Comment (stutisk) [reply components/badges/badges.html]: yeah, actually i had to change it ,thanku!

File: components/badges/badges.html (hunk 1 above)
Comment (stutisk) [reply components/badges/badges.html:12]: yah this can be done ! thanku

File: components/badges/badges.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    content: "";
+    font-size: var(--size-1);
+    display: inline-block;
+    position: relative;
+    right: 20px;
+    bottom: 1px;
+    text-align: center;
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
```
Comment (RakshithBhat03) [inline components/badges/badges.css]: "border-radius" is defined twice. You can remove one of them.

**Review by RakshithBhat03 — APPROVED** [review]
Everything else looks good

File: components/badges/badges.css (hunk 5 above)
Comment (stutisk) [reply components/badges/badges.css]: removed , thanku for pointing it out

File: components/badges/badges.css (hunk 3 above)
Comment (stutisk) [reply components/badges/badges.css]: done!

**Pull request merged**